pipenv/utils.py

```diff
@@ -760,7 +764,7 @@ def repository(self):
             from pipenv.patched.piptools.repositories.pypi import PyPIRepository
             self._repository = PyPIRepository(
                 self.pip_args, use_json=False, session=self.session,
-                build_isolation=self.pip_options.build_isolation
+                build_isolation=False
```

Comment:
Any reason behind this?

Reply:
oh hrm, no... I can't even remember why I did this now